flake8diff/__init__.py

```diff
@@ -1,3 +1,7 @@
 __version__ = '0.2.0'
+__description__ = (
```

Comment:
moved short description here so that it can be reused both by `setup.py` and in `main.py`